portal-frontend/src/app/features/application-details/parcel/parcel.component.html

```diff
@@ -72,25 +74,34 @@ <h4>
         Invalid Format
       </app-validation-error>
     </div>
-    <div class="subheading2 grid-1">
-      PIN (Optional)<app-updated-banner *ngIf="updatedFields.includes(parcelInd + '.pin')"></app-updated-banner>
-    </div>
-    <div class="grid-double">
-      {{ parcel.pin }}
-      <app-no-data [showRequired]="showErrors && !!parcel.validation?.isPinRequired" *ngIf="!parcel.pin"></app-no-data>
-    </div>
-    <div class="subheading2 grid-1">
-      Purchase Date
-      <app-updated-banner *ngIf="updatedFields.includes(parcelInd + '.purchasedDate')"></app-updated-banner>
-    </div>
-    <div class="grid-double">
-      {{ parcel.purchasedDate | date }}
-      <app-no-data
-        [showRequired]="showErrors && !!parcel.validation?.isPurchasedDateRequired"
-        *ngIf="!parcel.purchasedDate"
-      >
-      </app-no-data>
-    </div>
+    <!--  Remove field if fee simple selected  -->
+    <ng-container *ngIf="parcel.ownershipType?.code === 'CRWN'">
```

Comment:
I'm not sure just hiding the field will be sufficient, if a user selects crown land then fills a pin then hits save, then toggles to fee simple the PIN would stay saved and still be present in the view page, PDF, ALCS, etc. You either need to add this ngIf to all of the above or clear it from the DB.